templates/wizards/projects/asn1acn/file.pro

```diff
@@ -0,0 +1,13 @@
+TEMPLATE = app
+CONFIG += console
+CONFIG -= app_bundle
+CONFIG -= qt
+
+%{JS: if ('%{AddAsnFile}' === 'true' && '%{AddAcnFile}' === 'true')
+        'DISTFILES += \\\\ \n %{AsnFullFileName} \\\\ \n %{AcnFullFileName}'
```

Comment:
ok, but it will *probably* change with implementing asn1scc support for qmake. Probably :)